src/actions/targetActions.js

```diff
@@ -20,6 +20,10 @@ export const createTarget = createThunk('CREATE_TARGET', async target => {
   return data;
 });
 
+export const deleteTarget = createThunk('DELETE_TARGET', async target => {
+  await targetService.deleteTarget(target);
+});
```

Comment:
I think this shoudl work:

```suggestion
export const deleteTarget = createThunk('DELETE_TARGET', targetService.deleteTarget);
```